edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/publisher/EdgeRuntimeDiagnosticsUpload.cs

```diff
@@ -14,6 +14,9 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Publisher
 
     public sealed class EdgeRuntimeDiagnosticsUpload : IMetricsPublisher
     {
+        // max message size is 256KB, keep a 1KB buffer for safty
+        const int MaxMessageSize = 255000;
```

Comment:
Isn't the 256KB max really 256 * 1024 = 262,144 bytes? So that's nearly a 7KB buffer...